.gitignore

```diff
@@ -344,6 +344,7 @@ sandbox
 
 #IL linker for testing
 linker
+src/tools/linker/test/Mono.Linker.Tests/TestResults.xml
```

Comment:
Can we set the intermediate path pointing to `artifacts/obj` root, so TestResults are not produced in the source tree?